src/tests/multihost/ad/test_dyndns.py

```diff
@@ -98,6 +98,7 @@ def remove_interface():
 @pytest.mark.usefixtures("reverse_zone", "disable_dns_forwarders", "change_client_hostname")
 @pytest.mark.dyndns
 @pytest.mark.tier2
+@pytest.mark.c_ares
```

Comment:
I would rather see it on a few of the individual tests. Instead of the whole tier2 test suite.